pkg/services/pluginsintegration/plugincontext/plugincontext.go

```diff
@@ -43,7 +42,7 @@ type Provider struct {
 func (p *Provider) Get(ctx context.Context, pluginID string, user identity.Requester, orgID int64) (backend.PluginContext, error) {
 	plugin, exists := p.pluginStore.Plugin(ctx, pluginID)
 	if !exists {
-		return backend.PluginContext{}, ErrPluginNotFound
+		return backend.PluginContext{}, plugins.ErrPluginNotRegistered
```

Comment:
Should use `.Errorf` to create an actual error that includes an error message.

Reply:
(not needed if https://github.com/grafana/grafana/pull/74928 goes as it is atm)